### ISSUE TITLE
`rename` should not require the field to be listed in `accept`

### ISSUE THREAD

**Issue by vitiral** [malloydata/malloy#1288]
### What happens?

You seem to need to specify the field twice, both in `accept:` and in `rename:`

### To Reproduce

```
source: groundhogs is table('duckdb:data/groundhogs.csv') extend {
    primary_key: the_day
    accept: the_day, lug, shortname, city, region,
            isGroundhog,
           -- shadow --   <---- this line is required
    rename: saw_shadow is shadow
}
```

### OS:

Linux

### Malloy Client:

local VSCode

### Malloy Client Version:

latest VS Code

### Database Connection:

DuckDB

**Comment by lloydtabb** [comment]
Some SQL languages make `accept` and `rename` part of the same statement.  Perhaps this is better as some kind of blocking syntax?  @mtoy-googly-moogly putting a bug in your ear.

**Comment by vitiral** [comment]
Another pain point related to this: you need to `accept` a field to use it in `dimension` and `where`, even when you don't want it in the output schema. This requires me to make an extra `query` to get the data in the shape where it's ergonomic to view.

**Comment by lloydtabb** [comment]
Maybe `private:`. ...

**Comment by vitiral** [comment]
`private` would be nice for removing a sub-calculation, but I'm referring to something like this

```
source: mySource is ducdb.table('whatever.csv') extend {
  accept: a, b, c
  aggregate: sum_d is sum(d)   --   <-- fails because `d` is not in accept
}
```

**Comment by lloydtabb** [comment]
Yes, at some point do something like this.  `public`, `protected`, `private` is the way it is handled in OOP, often.  We will get to something like this when it makes it to the top of the list.
